src/openforms/registrations/contrib/objects_api/checks.py

```diff
@@ -15,17 +15,24 @@
 
 def check_objects_service():
     with get_objects_client() as client:
-        client.get("objects", params={"pageSize": 1})
+        resp = client.get("objects", params={"pageSize": 1})
+        resp.raise_for_status()
+        if not 200 <= resp.status_code < 300:
+            raise InvalidPluginConfiguration(_("Missing API credentials"))
 
 
 def check_objecttypes_service():
     with get_objecttypes_client() as client:
-        client.get("objecttypes", params={"pageSize": 1})
+        resp = client.get("objecttypes", params={"pageSize": 1})
+        resp.raise_for_status()
+        if not 200 <= resp.status_code < 300:
+            raise InvalidPluginConfiguration(_("Missing API credentials"))
```

Comment:
there's a missing test :)

Reply:
:expressionless:

Reply:
Implementation is flawed in my opinion because tests rely on the order of checks, which makes it annoying to test

Reply:
setting it up is also annoying, so testing/checking makes it annoying too

the reality is that the plugin as a whole is validated and multiple aspects require correct configuration for that. The tests should really be set up as all relevant permutations: one service is not properly configured, all the rest are -> the check should highlight something is wrong. If the "other services" are set up correctly, the order aspect becomes irrelevant.